runtime/j9vm/exports.cmake

```diff
@@ -446,6 +443,26 @@ else()
 	)
 endif()
 
+if(JAVA_SPEC_VERSION LESS 23)
+	if(NOT JAVA_SPEC_VERSION LESS 15)
+		jvm_add_exports(jvm
+			JVM_IsCDSDumpingEnabled
+		)
+		if(NOT JAVA_SPEC_VERSION LESS 16)
+			jvm_add_exports(jvm
+				JVM_IsDumpingClassList
+				JVM_IsSharingEnabled
+			)
+		endif()
+	endif()
+endif()
+
```

Comment:
See the comment near the removal of `JVM_IsCDSDumpingEnabled`.

Reply:
Removed